core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -270,63 +273,71 @@ fun LPGenericElevatedCard(
  */
 @Composable
 fun LPGenericElevatedCardImage(
```

Comment:
Please add the modifier like parameter and define a default variable is it is not sending like this `modifier: Modifier = Modifier`